internal/c/libqb/include/image.h

```diff
@@ -0,0 +1,58 @@
+//----------------------------------------------------------------------------------------------------
+//    ___  ___   __ _ _  ___ ___   ___                       _    _ _
+//   / _ \| _ ) / /| | || _ \ __| |_ _|_ __  __ _ __ _ ___  | |  (_) |__ _ _ __ _ _ _ _  _
+//  | (_) | _ \/ _ \_  _|  _/ _|   | || '  \/ _` / _` / -_) | |__| | '_ \ '_/ _` | '_| || |
+//   \__\_\___/\___/ |_||_| |___| |___|_|_|_\__,_\__, \___| |____|_|_.__/_| \__,_|_|  \_, |
+//                                               |___/                                |__/
+//
+//  QB64-PE Image Library
+//  Powered by stb_image (https://github.com/nothings/stb) & dr_pcx (https://github.com/mackron/dr_pcx)
+//
+//  Copyright (c) 2022 Samuel Gomes
+//  https://github.com/a740g
+//
+//-----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+//-----------------------------------------------------------------------------------------------------
+// HEADER FILES
+//-----------------------------------------------------------------------------------------------------
+#include <stdint.h>
+#include <stdio.h>
+//-----------------------------------------------------------------------------------------------------
+
+//-----------------------------------------------------------------------------------------------------
+// MACROS
+//-----------------------------------------------------------------------------------------------------
```

Comment:
This is fine as-is, but I create #181 to look at improving our logging infrastructure and combining this with the similar audio logic you added.

Reply:
I am all in for this. We can make use of an existing library instead of writing our own. I'll research whenever I get some time.